fragdenstaat_de/fds_blog/templates/fds_blog/_article_detail.html

```diff
@@ -102,36 +100,34 @@ <h3 class="small text-secondary">
                                 {% trans "About the authors" %}
                             {% endif %}
                         </h3>
-                        <div class="row row-cols-1 row-cols-md-2 g-4">
+                        <div class="row row-cols-1 row-cols-md-2 g-3">
                             {% for author in authors_with_profiles %}
-                                <div class="col">
-                                    <div class="card h-100">
-                                        <div class="card-body">
-                                            <h4 class="card-title">{{ author.get_full_name }}</h4>
-                                            {{ author.user.profile_text | markdown }}
-                                        </div>
+                                <div class="col d-flex">
+                                    <div class="border border-2 border-gray-300 p-3 tight-margin">
+                                        <h4 class="fs-6">{{ author.get_full_name }}</h4>
+                                        {{ author.user.profile_text | markdown }}
                                     </div>
                                 </div>
                             {% endfor %}
                         </div>
                     </div>
-                    {% if authors_without_profiles %}
-                        <div class="container-lg mt-4">
-                            <h3 class="small text-secondary">
-                                {% if authors_without_profiles|length == 1 %}
-                                    {% trans "Further author" %}
-                                {% else %}
-                                    {% trans "Further authors" %}
-                                {% endif %}
-                            </h3>
-                            <ul class="list-unstyled d-md-flex flex-wrap gap-3">
-                                {% for author in authors_without_profiles %}<li class="mb-1">{{ author.get_full_name }}</li>{% endfor %}
-                            </ul>
-                        </div>
-                    {% endif %}
                 {% endif %}
-            {% endblock article_author_bios %}
-        </section>
+                {% if authors_without_profiles %}
+                    <div class="container-lg">
+                        <h3 class="small text-secondary">
+                            {% if authors_without_profiles|length == 1 %}
```

Comment:
Nitpick (since it's not relevant for the languages fragdenstaat_de supports at the moment): could be proper pluralization (`blocktranslate` + `plural`), https://www.gnu.org/software/gettext/manual/gettext.html#Plural-forms